### PR TITLE
Docs fix spelling issues

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I found and fixed several minor spelling issues.
Br, elias.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ table below shows how its core methods relate to the stages of a test run:
 |---------------|--------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | compilation   | `instrument` | A **pre-compilation** step: Rewrites contracts and generates an instrumentation data map.                                                                                              |
 | client launch |   `attachToHardhatVM`  | A **pre-test** step: Enables coverage collection enabled in a HardhatEVM client. As the client runs it will mark line/branch hits on the instrumentation data map.         |
-| test          | `report`     | A **post-test** step: Generates a coverage report from the data collected by the VM after tests complete. |
+| test          | `report`     | A **post-test** step: Generates a coverage report from the data collected by the VM after tests are complete. |
```
Comment (cgewecke) [inline docs/api.md:12]: @nnsW3 Apologies, I prefer the original here because it's shorter and makes sense - "complete" is being used as a verb here. 

Would it be possible to revert this change?

**Review by cgewecke — CHANGES_REQUESTED** [review]
Thanks again! Everything LGTM apart from comment about the api.md doc change...

**Comment by nnsW3** [comment]
done

**Review by cgewecke — APPROVED** [review]
Great, thank you. 💯
